python_modules/libraries/dagster-gcp/dagster_gcp/bigquery/resources.py

```diff
@@ -96,3 +101,39 @@ def bigquery_resource(context):
     bq_resource = BigQueryResource.from_resource_context(context)
     with bq_resource.get_client() as client:
         yield client
+
+
+def get_data_freshness(
+    context: OpExecutionContext,
```

Comment:
If we require keyword arguments, it will allow us to make `context` non-required in the future if we choose.

Reply:
good idea.